bug: alpha5 distribution fails with connectors

Reproduce the error:

- Download [alpha5 release](https://github.com/camunda/camunda-distributions/releases/tag/docker-compose-alpha)
- run `docker compose up -d`
- Go see logs of connectors

```
2025-03-13 13:32:09 Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'operateAuthentication' defined in class path resource [io/camunda/operate/spring/OperateClientConfiguration.class]: Failed to instantiate [io.camunda.operate.auth.Authentication]: Factory method 'operateAuthentication' threw exception with message: 'operate.client.profile' is required
2025-03-13 13:32:09     at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:657)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:489)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1351)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1181)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:563)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:523)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:336)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:288)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:334)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
2025-03-13 13:32:09     at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:254)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1626)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1514)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:913)
2025-03-13 13:32:09     at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:791)
2025-03-13 13:32:09     ... 133 common frames omitted
```

If I add

```
      - OPERATE_CLIENT_BASEURL=http://operate:8080
      - OPERATE_CLIENT_PROFILE=simple
```

then it works but I still see failures

```
2025-03-13 13:35:56 2025-03-13T12:35:56.262Z  WARN 1 --- [ult-executor-19] io.camunda.client.job.poller             : Failed to activate jobs for worker HTTP REST and job type io.camunda:http-json:1
2025-03-13 13:35:56 
2025-03-13 13:35:56 io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2025-03-13 13:35:56     at io.grpc.Status.asRuntimeException(Status.java:532)
2025-03-13 13:35:56     at io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:481)
2025-03-13 13:35:56     at io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:564)
2025-03-13 13:35:56     at io.grpc.internal.ClientCallImpl.access$100(ClientCallImpl.java:72)
2025-03-13 13:35:56     at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:729)
2025-03-13 13:35:56     at io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:710)
2025-03-13 13:35:56     at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
2025-03-13 13:35:56     at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:133)
2025-03-13 13:35:56     at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
2025-03-13 13:35:56     at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
2025-03-13 13:35:56     at java.base/java.lang.Thread.run(Unknown Source)
2025-03-13 13:35:56 Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: localhost/[0:0:0:0:0:0:0:1]:26500
2025-03-13 13:35:56 Caused by: java.net.ConnectException: Connection refused
2025-03-13 13:35:56     at java.base/sun.nio.ch.Net.pollConnect(Native Method)
2025-03-13 13:35:56     at java.base/sun.nio.ch.Net.pollConnectNow(Unknown Source)
2025-03-13 13:35:56     at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source)
2025-03-13 13:35:56     at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:336)
2025-03-13 13:35:56     at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:339)
2025-03-13 13:35:56     at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:776)
2025-03-13 13:35:56     at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
2025-03-13 13:35:56     at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
2025-03-13 13:35:56     at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
2025-03-13 13:35:56     at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
2025-03-13 13:35:56     at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
2025-03-13 13:35:56     at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
2025-03-13 13:35:56     at java.base/java.lang.Thread.run(Unknown Source)
```

Could you help?

--- a/docker-compose/versions/camunda-alpha/docker-compose.yaml
+++ b/docker-compose/versions/camunda-alpha/docker-compose.yaml
@@ -157,14 +157,18 @@ services:
     ports:
       - "8085:8080"
     environment:
-      - ZEEBE_CLIENT_BROKER_GATEWAY-ADDRESS=zeebe:26500
-      - ZEEBE_CLIENT_SECURITY_PLAINTEXT=true
+      - OPERATE_CLIENT_BASEURL=http://operate:8080
+      - OPERATE_CLIENT_PROFILE=oidc
+      - CAMUNDA_CLIENT_ZEEBE_GRPCADDRESS=http://zeebe:26500
+      - OPERATE_CLIENT_AUTHURL=http://keycloak:18080/auth/realms/camunda-platform/protocol/openid-connect/token
+      - OPERATE_CLIENT_AUDIENCE=operate-api
+      - OPERATE_CLIENT_CLIENTID=connectors
+      - OPERATE_CLIENT_CLIENTSECRET=XALaRPl5qwTEItdwCMiPS62nVpKs7dL7
       - ZEEBE_CLIENT_ID=${ZEEBE_CLIENT_ID}
       - ZEEBE_CLIENT_SECRET=${ZEEBE_CLIENT_SECRET}
       - ZEEBE_CLIENT_CONFIG_PATH=/tmp/zeebe_auth_cache
       - ZEEBE_TOKEN_AUDIENCE=zeebe-api
       - ZEEBE_AUTHORIZATION_SERVER_URL=http://keycloak:18080/auth/realms/camunda-platform/protocol/openid-connect/token
-      - CAMUNDA_OPERATE_CLIENT_URL=http://operate:8080
       - CAMUNDA_IDENTITY_ISSUER_URL=http://${KEYCLOAK_HOST}:18080/auth/realms/camunda-platform
       - CAMUNDA_IDENTITY_ISSUER_BACKEND_URL=http://keycloak:18080/auth/realms/camunda-platform
       - CAMUNDA_IDENTITY_CLIENT_ID=connectors

